rviz_rendering/CMakeLists.txt

```diff
@@ -126,6 +129,12 @@ if(TARGET Qt5::windeployqt)
     )
 endif()
 
+set(rviz_rendering_DIR "${CMAKE_CURRENT_SOURCE_DIR}/src/cmake")
+include(${CMAKE_CURRENT_SOURCE_DIR}/src/cmake/register_rviz_ogre_media_exports_hook-extras.cmake)
+
+install(DIRECTORY src/cmake
+        DESTINATION share/${PROJECT_NAME})
```

Comment:
Please don't do aligned indentations (c++ style check will catch this, but none for CMake). Instead do:

```cmake
install(DIRECTORY src/cmake
  DESTINATION share/${PROJECT_NAME})
```

Or:

```cmake
install(
  DIRECTORY src/cmake
  DESTINATION share/${PROJECT_NAME})
```

Please check the rest of the files, I'm seeing it a lot.